paper-checkbox.html

```diff
@@ -109,11 +109,14 @@
         }
       },
 
-      ready: function() {
-        if (Polymer.dom(this).textContent == '') {
+      attached: function() {
+        var trimmedText = Polymer.dom(this).textContent.trim();
+        if (trimmedText == '') {
```

Comment:
Loose equality checks on strings can sometimes cause unexpected behavior. I think it's fine in this case, but perhaps we should consider checking strictly?